tests/test_lib_tokens_sms.py

```diff
@@ -490,7 +490,6 @@ def test_21_failed_loading(self):
                                "HttpSMSProvider.HttpSMSProvider")
         c = token.create_challenge(transactionid)
         self.assertFalse(c[0], c)
-        self.assertTrue("Failed to load sms.providerConfig" in c[1], c[1])
```

Comment:
We should check that the correct message was logged. This can be don with the `log_capture()`decorator from the [`testfixtures`](https://testfixtures.readthedocs.io/en/latest/logging.html#the-decorator) package.